reference/ingestion-tools/fleet/install-standalone-elastic-agent.md

```diff
@@ -116,22 +144,19 @@ To install and run {{agent}} standalone:
 
     :::{tab-item} DEB
     
-    ::::{tip}
-    You must run this command as the root user because some integrations require root privileges to collect sensitive data.
-    ::::
+    TIP: You must run this command as the root user because some integrations require root privileges to collect sensitive data.
```

Comment:
```suggestion
```

The TIP doesn't make sense to DEB and RPM as the agent is already installed, and systemctl commands are always handled by root.